datasets/ph/most_wanted/crawler.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from typing import Dict
+import zavod.helpers as h
+from zavod import Context
+
+
+def crawl_row(context: Context, row: Dict[str, str]):
+    data = dict(row)
+    full_name = data.pop("name", None)
+    offense = data.pop("offense", None)
+    case_number = data.pop("case number", None)
+    source = data.pop("source", None)
+    reward = data.pop("reward", None)
+
+    entity = context.make("Person")
+    entity.id = context.make_id(full_name, case_number, reward)
```

Comment:
I don't think the reward is really needed here. The vast majority have case numbers so I think they're sufficiently unique. I would try and add more directly descriptive fields to ID for more uniqueness, e.g. offence before reward.